### PR TITLE
NDRS-925 block executor optimization (1/2)

### PR DESCRIPTION
**Pull request by dwerner**
This PR combines all messages and functionality from BlockExecutor into ContractRuntime, and represents step 1 of 2.

ref: https://casperlabs.atlassian.net/browse/NDRS-925

### CODE REVIEW

File: node/src/reactor/initializer.rs [hunk 1]
```diff
@@ -242,13 +262,35 @@ impl reactor::Reactor for Reactor {
...
+            event_queue: EventQueueHandle<Event>,
+            chainspec: Arc<Chainspec>,
+        ) -> Result<(Self, Effects<Event>), Error> {
+            let effect_builder = EffectBuilder::new(event_queue);
+            let (chainspec_loader, chainspec_effects) =
+                ChainspecLoader::new_with_chainspec(chainspec, effect_builder);
+            Self::new_with_chainspec_loader(config, registry, chainspec_loader, chainspec_effects)
+        }
+    }
+    impl NetworkedReactor for Reactor {
```
Comment (EdHastingsCasperLabs) [inline node/src/reactor/initializer.rs:286]: nitpick: separating blank line btwn impls, but could be corrected in 2/2

**Review by EdHastingsCasperLabs — APPROVED** [review]

**Review by fizyk20 — APPROVED** [review]
LGTM :+1:

**Comment by dwerner** [comment]
bors cancel

**Comment by dwerner** [comment]
This PR needs to be re-targeted towards a `dev` branch when we cut that.

**Pull request merged**